packages/@n8n/api-types/src/dto/tag/base-tag-request.dto.ts

```diff
@@ -0,0 +1,6 @@
+import { z } from 'zod';
+import { Z } from 'zod-class';
+
+export class BaseTagRequestDto extends Z.class({
```

Comment:
I'm wondering if we really need to create two separate DTOs here. Can we instead have a `CreateOrDeleteTagRequestDto` or `UpsertTagRequestDto` ?